src/components/Home.vue

```diff
@@ -637,4 +642,9 @@ export default {
 .test {
     z-index: 100;
 }
+.add-token-button {
+    z-index: 200;
+    font-size: 33px;
+    margin-left: 60px;
```

Comment:
Seeing from safari, `margin-left: 60px;` will show button in new line. Suggest to use `div` float right.
<img width="317" alt="Screenshot 2020-01-06 at 6 04 34 PM" src="https://user-images.githubusercontent.com/41148850/71811622-274daa00-30b0-11ea-8919-f96ebcb683bf.png">